### PR TITLE
DM-30300: exposurelog: change bleed deployment to base

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: services/exposurelog/values-base.yaml [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
+      - host: base.lsst.codes
         paths:
           - /exposurelog
     tls: []
     #  - secretName: chart-example-tls
     #    hosts:
     #      - chart-example.local
 
-vault_path: secret/k8s_operator/bleed.lsst.codes/postgres
+vault_path: secret/k8s_operator/base.lsst.codes/postgres
```
Comment (afausti) [inline services/exposurelog/values-base.yaml:23]: The correct value for the base ``vault_path`` is ``secret/k8s_operator/base-lsp.lsst.codes/postgres``

File: services/exposurelog/values-base.yaml [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
 
 pull-secret:
   enabled: true
-  path: secret/k8s_operator/bleed.lsst.codes/pull-secret
+  path: secret/k8s_operator/base.lsst.codes/pull-secret
```
Comment (afausti) [inline services/exposurelog/values-base.yaml:27]: Same as above.

**Review by afausti — COMMENTED** [review]
Small fix needed on the `vault_path` for the base environment.  You also need to enable `exposurelog` in `phalanx/science-platform/values-base.yaml`.  That should do it.

File: services/exposurelog/values-base.yaml (hunk 1 above)
Comment (r-owen) [reply services/exposurelog/values-base.yaml:23]: How about this entry:
```
    hosts:
      - host: base.lsst.codes
```
should that be base-lsp.lsst.codes as well?

Comment (r-owen) [reply services/exposurelog/values-base.yaml:23]: Looking at the summit file clearly it should be. I'll fix all three instances of base.lsst.codes.

**Review by afausti — APPROVED** [review]
Looks good!

**Pull request merged**